Loop/LoopBox.swift

```diff
@@ -42,6 +42,14 @@ internal class ScopedLoopBox<RootState, RootEvent, ScopedState, ScopedEvent>: Lo
             event: { [eventTransform] in eventTransform(event($0)) }
         )
     }
+
+    override func pause() {
+        root.pause()
```

Comment:
not sure about this approach, as this way one scoped LoopBox will have the "power" to pause the root (parent), and as a consequence all other sibling scoped loop boxes.